drivers/crc/Kconfig.stm32

```
@@ -0,0 +1,10 @@
+# Copyright (c) 2024 Brill Power Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+config CRC_STM32
+	bool "STM32 CRC driver"
+	default n
+	depends on DT_HAS_ST_STM32_CRC_ENABLED && CRC_HW
```

```suggestion
	depends on DT_HAS_ST_STM32_CRC_ENABLED
```
(comes implicitly from `Kconfig` which `source`s this - see https://docs.zephyrproject.org/latest/build/kconfig/tips.html#lack-of-conditional-includes)